psl-evaluation/src/main/java/org/linqs/psl/utils/evaluation/statistics/DiscretePredictionComparator.java

```diff
@@ -78,7 +78,7 @@ public void setResultFilter(AtomFilter af) {
 	@Override
 	public DiscretePredictionStatistics compare(Predicate p) {
 		countResultDBStats(p);
-		//System.out.println("tp " + tp + " fp " + fp + " tn " + tn + " fn " + fn);
+        //System.out.println("TEST:\ntp " + tp + " fp " + fp + " tn " + tn + " fn " + fn);
```

Comment:
Remove debugging prints like this. (I know it was there earlier, but it shouldn't be. In generally debugging statements (outside of log.debug()) should not get pushed.)